README.md

```diff
@@ -0,0 +1,35 @@
+## React Tutorial from Scratch– A Step-by-Step Guide (2021 Update)
+
+This repo holds all the code for a React todos application. You can follow the step by step [guide to building the awesome project here](https://ibaslogic.com/react-tutorial-for-beginners/).
+
+## [Click to See the Todos App](https://ibaslogic.github.io/react-todo-project/)
+
```

Comment:
- [x] Please kindly ensure that the live version actually works because at the moment it does not.